packages/common/infra/src/atom/settings.ts

```diff
@@ -30,6 +31,7 @@ const appSettingBaseAtom = atomWithStorage<AppSetting>('affine-settings', {
   autoCheckUpdate: true,
   autoDownloadUpdate: true,
   enableTelemetry: true,
+  enableSnapshotImportExport: false,
```

Comment:
move this flag to https://github.com/toeverything/AFFiNE/blob/8b4511e4c8e19c440fcbb99a53975eedf7d4610c/packages/common/infra/src/modules/feature-flag/constant.ts#L8